packages/calcite-components/src/components/table-header/table-header.tsx

```diff
@@ -74,7 +74,7 @@ export class TableHeader extends LitElement implements LoadableComponent {
    * @notPublic
    */
   /** TODO: [MIGRATION] This component has been updated to use the useT9n() controller. Documentation: https://qawebgis.esri.com/arcgis-components/?path=/docs/references-t9n-for-components--docs */
-  messages = useT9n<typeof T9nStrings>();
+  messages = useT9n<typeof T9nStrings>({ blocking: true });
```

Comment:
Needed to block these to ensure an a11y test passes (https://dequeuniversity.com/rules/axe/4.10/empty-table-header?application=axeAPI).